web/handlers/admin/notices.go

```diff
@@ -156,5 +156,5 @@ func (nh noticeHandler) save(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	http.Redirect(rw, req, redirect, http.StatusTemporaryRedirect)
+	http.Redirect(rw, req, redirect, http.StatusSeeOther)
```

Comment:
I did this because I wanted the edit notice page to go to (upon saving) the `complete:notice:show` page, and that was originally a POST request, and the notice show is for GET, so I used HTTP 303.

Reply:
I need to read up on this, honestly. I can't remember with of the 3xx was the _don't cache me_ one. Was afraid to break behavior after save in unexpected ways.